tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "useUnknownInCatchVariables": false,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Not entirely sure what this does but things were red/not resolving in my IDE then I clicked a button and it added this which fixed it lmao